test/common/formatter/substitution_formatter_test.cc

```diff
@@ -2092,6 +2095,113 @@ TEST(SubstitutionFormatterTest, FilterStateFormatter) {
   }
 }
 
+TEST(SubstitutionFormatterTest, ProxyProtocolTlvsFormatter) {
```

Comment:
super nit: is this test ordered in the same way among other formatters as in the .cc file?

Reply:
I put this test between the `FilterStateFormatter` tests and the `DownstreamPeerCertVStartFormatter` tests to mirror the ordering of the formatters [here](https://github.com/10gen/envoy-serverless/blob/3b7b3dcabc193a29dff50b521b502c2f25596c08/source/common/formatter/substitution_formatter.cc#L464-L474) in `substitution_formatter.cc`. Does something seem off to you about this ordering?

Reply:
SGTM. Just double checking.